.devcontainer/Dockerfile

```diff
@@ -8,7 +8,7 @@ USER root
 RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash
 
 RUN apt-get install -qq nodejs npm libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev \
-    libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb curl git jq ssh
+    libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb curl jq ssh
```

Comment:
Similar to my note on Dockerfile about the short-term need to install git in the app container, I'd love to know if others are ok with change here. We would just need to remember to put this back later.

Reply:
Noted in #612 that we need to put this back later.